CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+- Merge behaviors of user/channel types and add support for `forHandle` (#339, fix for #338)
+  - `--type user` is now **DEPRECATED**
```

Comment:
Remember the empty line after the header.

I believe a `### Deprecated` section would be more visible and practical.